src/App.tsx

```diff
@@ -1,73 +1,39 @@
 import React from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Alert } from './components/alert/Alert';
+import { Dropdown } from './components/dropdown/Dropdown';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = React.useState<Person | null>(
+    null,
+  );
+  const [appliedQuery, setAppliedQuery] = React.useState('');
+  const filteredPeople = React.useMemo(() => {
+    return peopleFromServer.filter(person =>
+      person.name.toLowerCase().includes(appliedQuery.toLowerCase()),
+    );
+  }, [appliedQuery]);
+  const isEmptyPeopleList = filteredPeople.length === 0;
```

Comment:
it doesn't seem that you need this variable. The nested components already have access to the variable `people`, so they can check whether it is empty on their own